.github/workflows/actions.yml

```diff
@@ -1,109 +1,16 @@
-          ./build-wasm.sh
-          ./wasm-opt.sh
-      - name: Run VM tests
-        run: |
-          source .github/workflows/env
-          cargo test --features elrond-wasm-debug/mandos-go-tests
-      - name: Generate file size report
-        run: |
-          source .github/workflows/env
-          ./sizes.sh > sizes.txt
-      - name: Upload file size report
-        uses: actions/upload-artifact@v2
-        with:
-          name: sizes
-          path: sizes.txt
-      - name: Download base report
-        uses: dawidd6/action-download-artifact@v2
-        if: ${{ github.event_name == 'pull_request' }}
-        continue-on-error: true
-        with:
-          workflow: actions.yml
-          name: sizes
-          commit: ${{github.event.pull_request.base.sha}}
-          path: base-sizes
-      - name: Generate report
-        id: sizes-report
-        if: ${{ github.event_name == 'pull_request' }}
-        run: |
-          source .github/workflows/env
-          python tools/size-report/size_report.py base-sizes/sizes.txt sizes.txt --allow-missing > report.md
-      - name: Render report
-        id: template
-        uses: chuhlomin/render-template@v1.2
-        if: ${{ github.event_name == 'pull_request' }}
-        with:
-          template: report.md
-          vars: |
-            base: ${{github.event.pull_request.base.sha}}
-            head: ${{github.event.pull_request.head.sha}}
-      - name: Find sizes report comment
-        id: fc
-        uses: peter-evans/find-comment@v1
-        if: ${{ github.event_name == 'pull_request' }}
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          comment-author: 'github-actions[bot]'
-          body-includes: Contract file size comparison
-      - name: Create or update sizes report comment
-        uses: peter-evans/create-or-update-comment@v1
-        if: ${{ github.event_name == 'pull_request' }}
-        with:
-          comment-id: ${{ steps.fc.outputs.comment-id }}
-          issue-number: ${{ github.event.pull_request.number }}
-          body: ${{ steps.template.outputs.result }}
-          edit-mode: replace
-  rust_test:
-    name: Rust tests
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          default: true
-          toolchain: nightly
-      - name: Run rust tests
-        run: cargo test
-  clippy_check:
-    name: Clippy linter check
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-            toolchain: nightly
-            components: clippy
-            default: true
-      - uses: actions-rs/clippy-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-
+  contracts:
+    name: Contracts
+    uses: ElrondNetwork/elrond-actions/.github/workflows/contracts.yml@v1
+    with:
+      rust-toolchain: nightly-2021-11-04
```

Comment:
The release workflow uses the docker image `elrondnetwork/elrond-sdk-erdpy-rust:frozen-003`, which has:

```
LABEL erdpy=1.2.2
LABEL rust=nightly-2022-03-01
LABEL wasm-opt=v1.3.0
LABEL wasm-opt-binaryen=version_105
LABEL nodejs=v12.18.3
```

Therefore, perhaps use rust `nightly-2022-03-01` (for consistency)?